src/controllers/user.go

```diff
@@ -2,22 +2,23 @@ package controllers
 
 import (
 	"encoding/json"
-	"github.com/gin-gonic/gin"
 	"log"
 	"minitwit/src/database"
 	"minitwit/src/functions"
 	"minitwit/src/models"
 	"net/http"
+
+	"github.com/gin-gonic/gin"
 )
 
 func userHandlers(router *gin.Engine) {
 
-	router.GET("/:user/follow", func(c *gin.Context) {
+	router.POST("/:user/follow", func(c *gin.Context) {
```

Comment:
This should be GET requests. according to the [Minitwit.py](https://github.com/DevelOpsITU/MiniTwit/blob/48dcde50e0360203d7e278c7fc85f180c54cb384/minitwit.py#L140)

Where absent request method means GET.